python/ray/rllib/a3c/shared_model.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+from ray.rllib.a3c.policy import (
+    categorical_sample, linear,
+    normalized_columns_initializer, Policy)
+
+from ray.rllib.models.catalog import ModelCatalog
+
+
+class SharedModel(Policy):
```

Comment:
Learning capabilities with the VisionNet Model and current hyperparameters is near none on Pong, but the FCNet model works on basic control tasks